data/remnant/remnant missions.txt

```diff
@@ -2231,3 +2243,659 @@ mission "Remnant: Face to Maw 2B"
+		personality mute
+		ship "Ember Waste Node" "Unknown Entity"
+	on enter Peragenor
+		dialog `The creature comes to a stop in front of you, spins around briefly, then spreads out its tentacles in another pattern. Following the Remnant's intuition, you match it to your star charts as probably being Convector. After holding the position for a minute, the creature realigns itself and starts to shimmer with the tell-tale signs of starting to make a jump, but it holds it, as if waiting for you.`
+	on enter Convector
+		dialog `It seems the analysis of the location was right. The creature quickly approaches you and repeats its previous dance, ending up with a slightly different frozen position. From this vantage, you suspect that it is probably Nenia. This time, as you watch it more closely, it looks like it is following in your wake somehow.`
+	on enter Nenia
+		dialog `You have never seen such activity from the void sprites. Many quickly congregate around the strange ship and seem to escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the Archon methodically pulverizing a Korath Raider.`
+	on complete
+		event "remnant: nenia ember waste node"
+		conversation
+			`You follow the creature down through the clouds of Nasqueron and emerge into what can only be described as an amphitheater in the clouds. Thousands of void sprites fill the area, and the creature you have been following floats near the center of it. Your external sensors pick up myriad pulses of energy and eerie whistling noises that seem to resonate in the clouds. As the <ship> moves forward, there is a sudden silence, and the flickers of energy synchronize.`
+			`	You feel a sudden pressure in your mind and a sense of presence as the world around you fades out somewhat.`
+			`	"Know you not, do I. Live creature inside not-quite creature, like ones who flew but not everywhere." You are momentarily disoriented as you look around and see hundreds of views of yourself peering out the cockpit window. "Your intention reverberates - Know and not know - Return with more echoes - Perhaps I shall hear."`
+			`	With that, the pressure vanishes, and you are left staring out the window at the rapidly dispersing flock of void sprites. You aren't entirely sure what the voice meant, but you are quite sure that the Remnant are going to want to hear about this.`
+
+
+
+event "remnant: nenia ember waste node"
+	system "Nenia"
+		add fleet "Ember Waste Node" 10000
+	
+
+
+mission "Remnant: Cognizance 17"
+	name "Report back to Aventine."
+	description "The Remnant will want to know about what happened on Nasqueron."
+	source "Nasqueron"
+	landing
+	destination "Aventine"
+	to offer
+		has "Remnant: Cognizance 16: done"
+	
+
+mission "Remnant: Cognizance 18"
+	name "Go collect some void sprites"
+	description "Escort the Pelican fleet to <waypoints> and back to <planet>."
+	source
+		government "Remnant"
+	landing
+	destination Aventine
```

Comment:
```suggestion
	destination Esquiline
```
This should be Esquiline, right?

Reply:
No, because to land on Esquiline the player would need to be flying something with gaslining. The mission explicitly tells the player they don't need to land in Nenia as a subtle hint they should probably fly something more combative than the puffin or (if unlocked, penguin) The player doesn't need to actually go to Esquiline, just Pantica (which is the system Esquiline is in); and then proceed to land on Aventine.

Reply:
Makes sense